core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -132,11 +127,11 @@ public static boolean testDBCallEquivalence(GraphDatabaseService db, String test
             testSet.set(gatherRecords(r));
         });
 
-        System.out.println("COMPARE TO:");
-        System.out.println(toCSV(compareTo.get()));
-
-        System.out.println("TEST SET:");
-        System.out.println(toCSV(testSet.get()));
+//        System.out.println("COMPARE TO:");
+//        System.out.println(toCSV(compareTo.get()));
+//
+//        System.out.println("TEST SET:");
+//        System.out.println(toCSV(testSet.get()));
```

Comment:
I think out-commented code is just slightly less bad than printlines in the code... Maybe at least add a comment saying something like `uncomment this to debug` so it is clear they are there on purpose.